docs/pages/access-controls/guides/webauthn.mdx

```diff
@@ -235,82 +242,411 @@ $ tsh login --proxy=mytenant.teleport.sh
   SSO users should configure multi-factor authentication in their SSO provider.
 </Admonition>
 
-## U2F
+## Configure per-session MFA
```

Comment:
For consistency with other how-to guides, and since this guide is a single flow of instructions, I suggest we make all H2 sections in the main body start with "Step n/d. ", and make the headings within these H2s unnumbered H3s.